R/plotPropselRange.R

```diff
@@ -0,0 +1,144 @@
+#' @importFrom graphics lines
+#' @importFrom lavaan cfa HolzingerSwineford1939
```

Comment:
I don't think you need to import the ones on this line here. It's only required when you call certain functions inside our functions